spec/cypress/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ["plugin:cypress/recommended"],
+  rules: {
+    "import/no-commonjs": ["off"],
+  },
+};
```

Comment:
Does this "stack" with the main file? Do those rules apply too?

Reply:
yep! eslint looks upwards until it finds a `root: true` and merges everything it finds

Reply:
Nice!